### PR TITLE
Drone.io integration for Weblinks

### PR DESCRIPTION
**Pull request by yvesh**
### Summary of Changes

Integration of drone.io into weblinks

### Testing Instructions

Review and check log of drone.io

//cc @puneet0191 @javigomez

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - ./.drone/build.sh
+    - apache2ctl start
+    - service mysql start
+    - cd /tests/www
+    - export DISPLAY=:0
+    - Xvfb -screen 0 1024x768x24 -ac +extension GLX +render -noreset > /dev/null 2>&1 &
+    - sleep 3
+    - fluxbox &
+    - vendor/bin/robo run:tests
+    - echo "Finished tests at $(date)"
```
Comment (810) [inline .drone.yml]: I don't think this is needed

Comment (yvesh) [inline .drone.yml]: Hey Jelle, we are testing and comparing the performance versus travis currently ;-) That's the reason i added it.

Comment (810) [inline .drone.yml]: i think the time is always displayed. So you don't need to add it

Comment (yvesh) [inline .drone.yml]: Okay removed it :-)

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  image: yveshoppe/joomla-systemtests:latest
+  commands:
+    - chmod a+x .drone/build.sh
+    - ./.drone/build.sh
+    - apache2ctl start
+    - service mysql start
+    - cd /tests/www
+    - export DISPLAY=:0
+    - Xvfb -screen 0 1024x768x24 -ac +extension GLX +render -noreset > /dev/null 2>&1 &
+    - sleep 3
```
Comment (javigomez) [inline .drone.yml]: not sure, but this type of errors use to happen in Travis too. We solved by increasing the sleep time giving more time Xvfb-fluxbox to launch and they dissapeared:

```
...
Failed to read: session.screen0.menu.alpha
Setting default value
Failed to read: session.screen0.menuDelay
Setting default value
...
```

**Comment by javigomez** [comment]
Tests passing at http://82.165.166.30/joomla-extensions/weblinks/5. 
Merging.

**Pull request merged**